src/blocks/music.rs

```diff
@@ -94,7 +94,7 @@
 //! action = "play_pause"
 //! ```
 //!
-//! Scroll to change the player volume, use the forward and back buttons to seek:
+//! Scroll to change the player volume, use the forward and backward buttons to seek:
```

Comment:
For some reason, `back` is the actual name of that button, so I think it's better to leave this one.

Reply:
Okay, then maybe this is a separate naming issue we should discuss another time. For now, I will revert his change.